marvel_superheroes/src/store/actionTypes/types.tsx

```diff
@@ -0,0 +1,15 @@
+export const SET_HEROES_NAME_START_WITH = 'SET_HEROES_NAME_START_WITH'
```

Comment:
You should have only 3 actions for each backend request. For example, GET_HERO_REQUEST, GET_HERO_SUCCESS, GET_HERO_ERROR. Rewrite all these actions